addons/website/static/src/snippets/s_table_of_content/000.js

```diff
@@ -68,15 +67,12 @@ const TableOfContent = publicWidget.Widget.extend({
         this.$target.css('top', isHorizontalNavbar ? position : '');
         this.$target.find('.s_table_of_content_navbar').css('top', isHorizontalNavbar ? '' : position + 20);
         position += isHorizontalNavbar ? this.$target.outerHeight() : 0;
-        if (this.previousPosition !== position) {
```

Comment:
So here the commit you revert creates a bug in 16.0 but not in previous versions where it was merged?

Reply:
No because the mobile preview was not available before 16.0. But as we will finally patch the lib, I will not have to disable the scrollspy and so we can keep the commit :+1: